ui/public/vscode.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <title>Kestra - VSCode</title>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+</head>
+<body>
+<script src="vscode-web/dist/out/vs/loader.js"></script>
+<script src="vscode-web/dist/out/vs/webPackagePaths.js"></script>
+<script>
+    const queryParams = Object.fromEntries(
+        window.location.search.substring(1)
+            .split("&")
+            .map(str => str.split("="))
+    );
+    const KESTRA_UI_PATH = queryParams["KESTRA_UI_PATH"];
+    const KESTRA_BASE_PATH = queryParams["KESTRA_BASE_PATH"];
+    const THEME = queryParams["THEME"];
+    Object.keys(window.webPackagePaths).map(function (key) {
+        window.webPackagePaths[key] = `${window.location.origin}${KESTRA_UI_PATH}vscode-web/dist/node_modules/${key}/${window.webPackagePaths[key]}`;
+    });
+    require.config({
+        baseUrl: `${window.location.origin}${KESTRA_UI_PATH}vscode-web/dist/out`,
+        recordStats: true,
+        trustedTypesPolicy: window.trustedTypes?.createPolicy("amdLoader", {
+            createScriptURL(value) {
+                return value;
+            }
+        }),
+        paths: window.webPackagePaths
+    });
+    window.product = {
```

Comment:
Is this copy/paste ? If not please write some comment for people that will work on it after you.
If yes, please provide the source or a documentation link